### ISSUE TITLE
MAX_RAM is not applied because of sed.

### ISSUE THREAD

**Issue by nicezic** [Renegade-Master/zomboid-dedicated-server#58]
**Describe the bug**
MAX_RAM is not applied because of sed mistake.

in run_server.sh

>      # Set the maximum amount of RAM for the JVM
>      sed -i "s/-Xmx.*/-Xmx${MAX_RAM}\",/g" "${SERVER_VM_CONFIG}"
> 

**To Reproduce**
always... 

**Expected behavior**
in ProjectZomboid64.json

>                 "-Xmx=8g",

**Screenshots**
in ProjectZomboid64.json

>                 "-Xmx8g",

**Desktop (please complete the following information):**
 - OS: win10
 - Docker Version v4.16.3
 - latest, master

**Additional context**

**Comment by Renegade-Master** [comment]
Hello!

I'm not sure that this is actually an issue.
According to the [Oracle docs](https://docs.oracle.com/cd/E13150_01/jrockit_jvm/jrockit/jrdocs/refman/optionX.html#wp999528), there can be a separator, but it does not appear to be required.

In [some](https://stackoverflow.com/a/14763095/10884550) [other](https://www.baeldung.com/jvm-parameters#bd-explicit-heap-memory---xms-and-xmx-options) [places](https://www.javatpoint.com/java-xmx) that I found, there seems to be several ways that the value can be specified.

Have you observed the memory usage of the Java executable exceeding the configured value?

**Comment by nicezic** [comment]
@Renegade-Master My mistake sorry.

**Comment by Renegade-Master** [comment]
@nicezic 

No problem 🙂
Thank you for the suggestion anyway. As far as I can tell, everything is working as expected, but I might be wrong. It's difficult to find out for sure without user feedback like this, so thank you.

**Comment by nicezic** [comment]
@Renegade-Master Thanks a lot for your understaing.
BTW... I have a good news.

**My assumption is only if Xms used, Xmx works as we expacted.** 

I'm using my own docker build to run the dedicated server.

Previously, in order to apply Xmx, I used  binding the json file, and copy it to target dir before starting up.
At that time, I used `-Xms=4g  -Xms=12g` mixed, and my server was running smoothly.
I saw my container's runing with up to 12GB memory usage.

A few days ago, I refered your implementation to apply the json file.
This time, accordingly `-Xmx12g` was the only option.
I had experienced laggy situations fighting many zombies and especially driving.
At the moment, I saw my docker container's mem stat. it say only Fxxx 3.9xGB !!!!

That's why I opened the wrong issue about not applied Xmx option.

Today, I tested `-Xms6g  -Xms12g` options.
**The container uses 6G memory as soon as started.**

So I recommand you test using 
```
Xms4g, Xmx$MAX_RAM 
Xms$MAX_RAM Xmx$MAX_RAM 
```

I personlly preper this one, `-Xms4g, -Xmx$MAX_RAM.`.
